cmake/modules/GoogleTest.cmake

```
@@ -32,7 +34,7 @@ ExternalProject_Add(
   #            -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE:PATH=ReleaseLibs
   #            -Dgtest_force_shared_crt=ON
   CMAKE_ARGS -G ${CMAKE_GENERATOR}
-                -DCMAKE_BUILD_TYPE=Release
+                -DCMAKE_BUILD_TYPE=${gtestbuild}
```

```suggestion
                -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

It's better for unix as well.

---

I just assumed google test was slow to run when built in debug config so it was always set to release

---

Probably you are right but maybe we should be consistent across platforms...